dev-haskell/hashable/hashable-1.3.1.0.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.7.9999
+#hackport: flags: sse41:cpu_flags_x86_sse4_1,sse2:cpu_flags_x86_sse2,integer-gmp:gmp
```

Comment:
We can remove all the unused flag maps: `sse41:cpu_flags_x86_sse4_1,sse2:cpu_flags_x86_sse2`

Reply:
I don't understand how it works so i left it as it was in the previous version of hashable, except the default that is mentioned in https://github.com/gentoo-haskell/hackport/issues/97